app/views/downloads/download_windows.html.erb

```diff
@@ -0,0 +1,67 @@
+<%- @section = "downloads" %>
+<%- @subsection = "" %>
+<%- @page_title = "Git - Downloading Package" %>
+
+<% content_for :sidebar do %>
+  <%= render 'shared/book' %>
+<% end %>
+
+<div id="main">
+  <h1>Downloading Git</h1>
+  <div class="callout downloading">
+    <h3>Your download is starting...</h3>
+    <p>
+      <%= raw "You are downloading the latest (<strong>#{@download.version.name}</strong>) <strong>#{@bitness}</strong> version of <strong>Git for Windows</strong>. This is the most recent <a href='#{@project_url}'>maintained build</a>. It was released <strong>#{time_ago_in_words @download.release_date} ago</strong>, on #{@download.release_date.strftime("%Y-%m-%d")}." %>
+    </p>
+    <p>
+      <%= raw "<strong>If your download hasn't started, <a href=\"#{@download.url}\">click here to download manually</a>.</strong>" %>
+    </p>
+    <h3>Other Git for Windows downloads</h3>
+    <h4>Git for Windows Setup</h4>
+    <p>
+      <%= raw "<strong><a href=\"#{@download32.url}\">32-bit Git for Windows Setup</a>.</strong>" %>
+    </p>
+    <p>
+      <%= raw "<strong><a href=\"#{@download64.url}\">64-bit Git for Windows Setup</a>.</strong>" %>
+    </p>
+    <h4>Git for Windows Portable ("thumbdrive edition")</h4>
+    <p>
+      <%= raw "<strong><a href=\"#{@download32portable.url}\">32-bit Git for Windows Portable</a>.</strong>" %>
+    </p>
+    <p>
+      <%= raw "<strong><a href=\"#{@download64portable.url}\">64-bit Git for Windows Portable</a>.</strong>" %>
+    </p>
+    <p class="small">
+      <%= raw "The current source code release is version <strong>#{@latest.name}</strong>. If you want the newer version, you can build it from <a href=\"#{@source_url}\">the source code</a>." %>
+    </p>
+    <iframe frameborder="0" height="1" src="<%= @download.url %>" width="1"></iframe>
+  </div>
+  <h2>Now What?</h2>
+  <p>
+    Now that you have downloaded Git, it's time to start using it.
+  </p>
+  <ul id="download-next-steps">
+    <li>
+      <a href="/book">
+      <img src="/images/icons/nav-read-book.png" />
+      <h3>Read the Book</h3>
+      <p>Dive into the Pro Git book and learn at your own pace.</p>
+      </a>
+    </li>
+    <li>
+      <a href="/downloads/guis">
+      <img src="/images/icons/nav-download-gui.png" />
+      <h3>Download a GUI</h3>
+      <p>Several free and commercial GUI tools are available for the #{@platform.capitalize} platform.</p>
+      </a>
+    </li>
+    <li>
+      <a href="/community">
+      <img src="/images/icons/nav-get-involved.png" />
+      <h3>Get Involved</h3>
+      <p>A knowledgeable Git community is available to answer your questions.</p>
+      </a>
+    </li>
+  </ul>
+</div>
+
```

Comment:
I wonder why we need a downloading and a downloading_windows, but that's another issue

Reply:
Yeah, could definitely be refactored at some point. I could add it to the project board.

Reply:
that's not that much of a CSS/redesign problem IMHO. we could just create an issue